File: 백준/Silver/1476. 날짜 계산/날짜 계산.kt
package org.example


fun main() {
    val br = System.`in`.bufferedReader()
    val (E, S, M) = br.readLine().split(" ").map { it.toInt() }
    var year = 1
    while (true) {
        if ((year - E) % 15 == 0 && (year - S) % 28 == 0 && (year - M) % 19 == 0) {
            println(year)
            return
        }
        year++
    }
    br.close()
}

File: 백준/Silver/1476. 날짜 계산/날짜 계산.py
import sys

input = sys.stdin.readline

year = 1
e, s, m = map(int, input().split())

while True:
    if (year - e) % 15 == 0 and (year - s) % 28 == 0 and (year - m) % 19 == 0:
        print(year)
        break
    year += 1
